spring-boot/admin/src/main/java/eu/coatrack/admin/model/repository/InternalProxyRepository.java

```diff
@@ -21,24 +21,18 @@
  */
 
 import eu.coatrack.api.Proxy;
-import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.data.repository.query.Param;
-import org.springframework.data.rest.core.annotation.RepositoryRestResource;
-import org.springframework.data.rest.core.annotation.RestResource;
-import org.springframework.security.access.prepost.PostAuthorize;
-import org.springframework.security.access.prepost.PostFilter;
 
-import java.util.List;
 import java.util.Optional;
 
 /**
  * These unsecured methods are only to be used within an authenticate() method. The lack of
```

Comment:
The comment is now a little bit confusing. 

Maybe better say that this is an additional repository for internal use, without using filter method security like the other repository for external access.

Reply:
I added your suggested text but still kept the remark about why this class exists at all. I find that important to know for readers and this purpose should be clear.

Reply:
: